compatimetrics.py

```diff
@@ -0,0 +1,501 @@
+import pandas as pd
+import numpy as np
+from sklearn.metrics import accuracy_score, mean_squared_error as mse
+
+
+def mean_squared_difference(pred1, pred2):
+    """ Calculates mean squared difference of two prediction vectors, which is
+    measure of similarity between two regression models prediction.
+
+    Parameters
+    ----------
+    pred1 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+    pred2 : list, numpy.array, pandas.series
+        numeric vector representing results of model prediction
+
+    Returns
+    -------
+    Numeric value
+        MSD value of given prediction pair
+    """
+    return mse(pred1, pred2)
```

Comment:
It seems to me that a function that only changes the name of the function is unnecessary and it would be better to just use mse in the code

Reply:
I'm not sure, but it seems to me that this changed name better explains what is going on it can be left.